cv-19-i-need-help-test/EndToEndTests/V2/GetHelpRequestTest.cs

```diff
@@ -0,0 +1,122 @@
+using System;
+using System.Collections.Generic;
+using Amazon.Lambda.APIGatewayEvents;
+using AutoFixture;
+using CV19INeedHelp.Boundary.V2;
+using CV19INeedHelp.Boundary.V2.Responses;
+using CV19INeedHelp.Data.V1;
+using CV19INeedHelp.Helpers.V2;
+using CV19INeedHelp.Models.V1;
+using CV19INeedHelpTest.TestHelpers;
+using FluentAssertions;
+using Newtonsoft.Json;
+using NUnit.Framework;
+
+namespace CV19INeedHelpTest.EndToEndTests.V2
+{
+    [TestFixture]
+    public class GetHelpRequestTest
+    {
+        private string _currentConnStr;
+        private Handler _handler;
+        private Fixture _fixture;
+        private Cv19SupportDbContext _dbContext;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _currentConnStr = Environment.GetEnvironmentVariable("CV_19_DB_CONNECTION");
+            const string connectionString = "Host=localhost;Database=i-need-help-test;Username=postgres;Password=mypassword";
+            _dbContext = new Cv19SupportDbContext(_currentConnStr ?? connectionString);
+            if (_currentConnStr == null) Environment.SetEnvironmentVariable("CV_19_DB_CONNECTION", connectionString);
+            _fixture = new Fixture();
+            CustomizeFixture.V2ResidentResponseParsable(_fixture);
+            _handler = new Handler();
+
+            AssertionOptions.AssertEquivalencyUsing(options =>
```

Comment:
Similarly, you can use a helper method for this as well (https://github.com/LBHackney-IT/cv-19-i-need-help-api/blob/282dd9596f438e7ab9c0f612d6dbef856b3850b3/cv-19-i-need-help-test/EndToEndTests/V2/GetHelpRequestsTest.cs#L28)

Reply:
Changes applied in #34